docs/operator/operations.md

```diff
@@ -0,0 +1,100 @@
+# Lifecycle Manager - Operations
+
+## Metrics
+
+Lifecycle Manager metrics are exposed over port `8080` through the metrics endpoint `/metrics`.
```

Comment:
As far as I know these metrics are not gigantic blobs, but partitioned using labels by SKR and module. Please explain the labels being used, their names, and their valuesets (like what SKR identifier you are using in the label, is it a shootid, runtimeid, subaccountid, tenantid, etc, we have like a dozen of these).

If there are metrics which are not partitioned by both SKRs _and_ modules, then add a column saying which metric has what labels for partitioning the data.